.github/workflows/upgrade_downgrade_test_semi_sync.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   upgrade_downgrade_test_e2e:
     timeout-minutes: 60
     name: Run Semi Sync Upgrade Downgrade Test
-    runs-on: gh-hosted-runners-16cores-1-24.04
+    runs-on: oracle-16cpu-64gb-x86-64
```

Comment:
Is there an alias that also guarantees a specific distro / version? We've been bitten many times before by floating names when they upgraded and we had to immediately rush and drop everything else to fix CI. 

We had explicit distro versions before so we could upgrade at a specific moment of our choice and now when these runner pools are updated. It also allows for updating in a separate PR while not being blocked anywhere else.

Reply:
> We didn't set up a specific alias for distro, but our intent is for this to ONLY mirror `ubuntu-latest` unless/until we hit a specific instance of needing an older ubuntu distro.

Right, but that's what explicitly caused us pain every time it was updated in the past. The biggest problem is not even that we don't want to upgrade, it's that it blocks us from even pro-actively upgrading to avoid the problems in the first place. So we have no way to deal with it and are forced to let things break and then have to rush to fix it 😢. 

So it's not that we need an older distro, it's that we need the explicit version so that we can control when to upgrade (which is often *before* `-latest` would change).

Reply:
@jeefy I don't want to hard block on this btw, but it's something that ideally we do have some solution for at some point. If that's more when Ubuntu 26.04 would be out next year that's also good but ideally there's something before then :smile:.

Reply:
I'll update our container builds/tags to ensure the runners can be pinned to specific os/versions. That'll come with a follow-up PR, probably post-KubeCon. 

Spit-balling, it'll probably be `oracle-16cpu-64gb-x86-64-24.04` vs `oracle-16cpu-64gb-x86-64`